preferencesfx/src/main/java/com/dlsc/preferencesfx/PreferencesFx.java

```diff
@@ -76,20 +56,17 @@ private void init() {
     // display initial category
     categoryController.setView(preferencesFxModel.getDisplayedCategory());
 
-    navigationView = new NavigationView(preferencesFxModel);
-    navigationPresenter = new NavigationPresenter(preferencesFxModel, navigationView);
+    NavigationView navigationView = new NavigationView(preferencesFxModel);
+    new NavigationPresenter(preferencesFxModel, navigationView);
```

Comment:
I don't think it's a great idea to create a new NavigationPresenter without a reference, this could lead to issues with the garbage collector collecting it as there's no reference to it